python/semantic_kernel/semantic_functions/chat_prompt_template.py

```diff
@@ -15,8 +16,15 @@
     from semantic_kernel.orchestration.sk_context import SKContext
 
 
+class ChatMessage(SKBaseModel):
```

Comment:
I'd like to see unit tests for adding, rendering, and manipulating chat message/message histories.

Reply:
agreed, this is def missing